subsys/bindesc/Kconfig.version

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BINDESC_DEFINE_VERSION
+	bool "Version binary descriptors"
+	help
+	  Add the version binary descriptors
+
+if BINDESC_DEFINE_VERSION
+
+config BINDESC_KERNEL_VERSION_STRING
+	bool "Kernel version string"
+	help
+	  The kernel version string, such as "3.4.0"
```

Comment:
consistency nitpick: this "3.4.0" is quoted but none of the other ones below are quoted